src/app/WriteClient.cpp

```diff
@@ -144,21 +144,23 @@ CHIP_ERROR WriteClient::ProcessWriteResponseMessage(System::PacketBufferHandle &
     return err;
 }
 
-CHIP_ERROR WriteClient::PrepareAttribute(const AttributePathParams & attributePathParams)
+CHIP_ERROR WriteClient::PrepareAttribute(const AttributePathParams & aAttributePathParams, DataVersion aDataVersion)
 {
     if (mState == State::Uninitialized)
     {
         ReturnErrorOnFailure(Init());
     }
-    VerifyOrReturnError(attributePathParams.IsValidAttributePath(), CHIP_ERROR_INVALID_PATH_LIST);
+    VerifyOrReturnError(aAttributePathParams.IsValidAttributePath(), CHIP_ERROR_INVALID_PATH_LIST);
```

Comment:
Shouldn't we also check that there is no data version if this is a wildcard (across endpoints) or group write?

Reply:
agree, we should do this check, are we going to support wildcard write?